csm/core/controllers/audit_log.py

```diff
@@ -50,7 +54,10 @@ async def get(self):
 
         start_date = request_data["start_date"]
         end_date = request_data["end_date"] 
-        return await self._service.get_by_range(component, start_date, end_date)
+        limit = request_data.get('limit')
```

Comment:
fields needs to be consistent with other paging rest api of CSM

Reply:
Done.